Gemfile

```diff
@@ -12,7 +12,7 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
   end
 end
 
-gem 'yard', '~> 0.9.12', :require => false
+gem 'yard', '~> 0.9.22', :require => false
```

Comment:
Can we limit this to 2.7.0 only until we resolve the docs differences? Unless you want to go through the diff now?

Reply:
You mean this one https://github.com/rspec/rspec-rails/pull/1938#issuecomment-570083773?

Reply:
Well you'd need to produce one for each repo :/